core-bundle/src/Resources/contao/models/PageModel.php

```diff
@@ -865,7 +865,7 @@ public function loadDetails()
 						$alias = $objParentPage->alias;
 						$name = $objParentPage->title;
 						$title = $objParentPage->pageTitle ?: $objParentPage->title;
-						$folderUrl = basename($alias) . '/' . $folderUrl;
```

Comment:
What was the reason there for `basename()`? An issue with directory traversal?

Reply:
Sadly, I do not remember. I only know that the `basename()` call has already been in the initial version of the `loadDetails()` method.

It cannot be a directory traversal issue though, because the alias is never used in a file system call.